{"created": 1627027907.5090663, "duration": 0.30499982833862305, "exitcode": 2, "root": "C:\\Users\\user", "environment": {"Python": "3.9.4", "Platform": "Windows-10-10.0.19041-SP0", "Packages": {"pytest": "6.2.2", "py": "1.10.0", "pluggy": "0.13.1"}, "Plugins": {"json-report": "1.4.0", "metadata": "1.11.0", "submit": "1.3.3"}}, "summary": {"total": 0, "collected": 0}, "collectors": [{"nodeid": "", "outcome": "failed", "result": [], "longrepr": "AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\runner.py:311: in from_call\n    result: Optional[TResult] = func()\nAppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\runner.py:341: in <lambda>\n    call = CallInfo.from_call(lambda: list(collector.collect()), \"collect\")\nAppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\main.py:690: in collect\n    for direntry in visit(str(argpath), self._recurse):\nAppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\pathlib.py:613: in visit\n    yield from visit(entry.path, recurse)\nAppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\pathlib.py:613: in visit\n    yield from visit(entry.path, recurse)\nAppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\pathlib.py:613: in visit\n    yield from visit(entry.path, recurse)\nAppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\pathlib.py:598: in visit\n    for entry in os.scandir(path):\nE   PermissionError: [WinError 5] \uc561\uc138\uc2a4\uac00 \uac70\ubd80\ub418\uc5c8\uc2b5\ub2c8\ub2e4: 'C:\\\\Users\\\\user\\\\AppData\\\\Local\\\\Application Data'"}], "tests": []}